coalib/parsing/InvalidFilterException.py

```diff
@@ -1,2 +1,8 @@
 class InvalidFilterException(Exception):
-    pass
+    def __init__(self, filter_name=None, available_filters=None):
+        self.filter_name = filter_name
+        self.available_filters = sorted(available_filters)
+        message = ('{!r} is an invalid filter. Available filters: {}'
```

Comment:
No need of extra parenthesis around the string.

Reply:
@aashutoshrathi @shreyans800755 Actually no need for intermediate `message` variable ;)